frontend/integration-tests/tests/kubevirt/vm.yaml.scenario.ts

```diff
@@ -1,16 +1,97 @@
 /* eslint-disable no-undef */
+import { execSync } from 'child_process';
 import { browser, ExpectedConditions as until } from 'protractor';
 
 import { appHost, testName } from '../../protractor.conf';
 import { isLoaded } from '../../views/crud.view';
+import * as yamlView from '../../views/yaml.view';
 import { removeLeakedResources } from './utils';
 import { detailViewAction, detailViewVmIcon, runningIcon } from '../../views/kubevirt/vm.actions.view';
+import { disksTab, nicsTab } from '../../views/kubevirt/virtualMachine.view';
 import Yaml from './models/yaml';
-
+import { VirtualMachine } from './models/virtualMachine';
+import { testNAD } from './mocks';
 
 describe('Test create vm from yaml', () => {
-  const leakedResources = new Set<string>();
+  const defaultVM = new Set<string>();
```

Comment:
isn't one set enough? What are the benefits of having two of them?

Reply:
Modified.